pkg/services/store/entity/db/dbimpl/dbimpl.go

```diff
@@ -69,7 +71,7 @@ func (db *EntityDB) GetEngine() (*xorm.Engine, error) {
 			}
 
 			connectionString := fmt.Sprintf(
-				"user='%s' password='%s' host='%s' port='%s' dbname='%s' sslmode='%s'", // sslcert='%s' sslkey='%s' sslrootcert='%s'",
+				"user=%s password=%s host=%s port=%s dbname=%s sslmode=%s", // sslcert='%s' sslkey='%s' sslrootcert='%s'",
```

Comment:
Had to change this so things can run locally. This issue is fixed [in this PR](https://github.com/grafana/grafana/pull/87656)